### ISSUE TITLE
Log-in status not working

### ISSUE THREAD

**Issue by thaJeztah** [docker/docs-base#10]
The logged-in status of the theme is no longer working, probably caused by changes in security-settings on https://hub.docker.com.

The logged-in status is handled via JavaScript, and shows/hides the log-in button if a `docker_sso_username` cookie is present in these lines: [theme/mkdocs/base.html#L153-L161](https://github.com/docker/docs-base/blob/e21aafe4258bade3fa06f98525243fdfd9a760d6/theme/mkdocs/base.html#L153-L161)

The `docker_sso_username` is set after a successful log-in on https://hub.docker.com/account/login/ (wrapped for readability);

    Set-Cookie:docker_sso_username=myusername; 
    Domain=.docker.com; expires=Sat, 18-Apr-2015 21:24:58 GMT; 
    httponly; Max-Age=86400; Path=/; secure

The problem is that the cookie is set as `httponly`, which [blocks access through JavaScript](https://www.owasp.org/index.php/HttpOnly). The theme is therefore not able to detect if the cookie is set.

Possible solution;
Remove the `httponly` restriction from the cookie. The cookie itself doesn't contain very sensitive data (only the username)

**Comment by moxiegirl** [comment]
@thaJeztah sounds like a security review is required 

ping @ewindisch

**Comment by thaJeztah** [comment]
@moxiegirl agreed, I can think of other solutions, but those would involve more changes to the Docker Hub, which makes it harder, because that's of course not open source.

**Comment by moxiegirl** [comment]
Not present in new site. Keeping open for old site fix.

**Comment by thaJeztah** [comment]
@moxiegirl do you still want to keep this open?
